api/src/main/java/org/openmrs/api/impl/AdministrationServiceImpl.java

```diff
@@ -201,6 +306,13 @@ public void updateGlobalProperty(String propertyName, String propertyValue) thro
 		if (gp == null) {
 			throw new IllegalStateException("Global property with the given propertyName does not exist" + propertyName);
 		}
+		
+		// if authenticated user is not supposed to edit globalProperty
```

Comment:
Does the above comment add any value?
Can you take a minute and look at this? https://improvingsoftware.com/2011/06/27/5-best-practices-for-commenting-your-code/